Library/ViewModels/SearchViewModel.swift

```diff
@@ -212,27 +220,16 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
       .filter { _, page in page == 1 }
       .map(first)
 
-    self.goToProject = Signal.combineLatest(self.projects, query)
-      .takePairWhen(self.tappedProjectProperty.signal.skipNil())
-      .map { projectsAndQuery, tappedProject in
-        let (projects, query) = projectsAndQuery
-
-        return (tappedProject, projects, refTag(query: query, projects: projects, project: tappedProject))
-      }
-
-    // Tracking
+    let sendEventForSearchResults = Signal.merge(
```

Comment:
I would consider renaming this given the name of the constant does not reflect its actual value. Seems like this is the searchResultsCount, only emits when a new query is searched for but does not actually fire the event.